lib/support.js

```diff
@@ -41,7 +39,7 @@ function isObject(obj, type) {
  * objects - An array of objects we want to test the type of
  * type - The expected type
  */
-function assertObject(objects, type) {
```

Comment:
Is these are going to throw an error on an issue, why is "assert" not the right word? http://en.wikipedia.org/wiki/Assertion_%28software_development%29